### ISSUE TITLE
Compilations woes with Ubuntu

### ISSUE THREAD

**Issue by gpatel-fr** [lavv17/lftp#703]
Hello

this is more a warning than an issue: I realized after a few hours of suffering than in the PRs and issues for this project there was more mentions of Fedora than Ubuntu when compiling it, so I spun a Fedora rawhide container and succeeded with the following incantations:
```
sudo dnf group install "C Development Tools and Libraries" "Development Tools"
sudo dnf install gperf gettext-devel ncurses-devel readline-devel openssl-devel
./autogen.sh --with-openssl --with-debug
make
```
the 'openssl' and 'debug' stanzas was because I wanted to debug the openssl version display, if the goal is to create a vanilla newx version there is no need and ./autogen.sh is sufficient.
So basically to succeed what is necessary is to use Fedora.

I probably could have downloaded the apt package source to solve it but the easiest way was the Fedora container.

Hope this helps someone.

I can of course submit a doc enhancement if it is felt useful.
